rust/ql/lib/codeql/rust/controlflow/ControlFlowGraph.qll

```diff
@@ -0,0 +1,36 @@
+/** Provides classes representing the control flow graph. */
+
+private import rust
+private import internal.ControlFlowGraphImpl
+private import internal.Completion
+private import internal.SuccessorType
+private import codeql.rust.controlflow.BasicBlocks
+import internal.Scope
```

Comment:
This exposes the `abstract` class `CfgScope`, which users could then in principle extend. So I think it would be better to do
```ql
private import internal.Scope as Scope

final class CfgScope = Scope::CfgScope;
```

Reply:
I've done as suggested. But I wonder if it would make sense to do it inside `Scope.qll` instead?